### ISSUE TITLE
Train the RPN

### ISSUE THREAD

**Issue by shanggdlk** [open-mmlab/mmdetection#1180]
Hi, May I know how to 1): fine-tune the pre-trained backbone using my own dataset on Mask-RCNN/faster-rcnnetc.; 2): pre-train the RPN using my own dataset on Mask-rcnn/faster-rcnn etc.?

Thank you!

**Comment by hellock** [comment]
Please refer to the [documentation](https://github.com/open-mmlab/mmdetection/blob/master/GETTING_STARTED.md#use-my-own-datasets).
